src/main/java/de/skuzzle/semantic/Version.java

```diff
@@ -25,10 +25,7 @@
 
 import java.io.ObjectStreamException;
 import java.io.Serializable;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Comparator;
-import java.util.List;
+import java.util.*;
```

Comment:
Revert the wildcard import
```suggestion
import java.util.ArrayList;
import java.util.Arrays;
import java.util.Comparator;
import java.util.List;
```